Cargo.toml

```diff
@@ -34,6 +34,7 @@ lazy_static = "0.1.15"
 regex = "0.1.71"
 rustc-serialize = "0.3"
 clippy-mini-macro-test = { version = "0.1", path = "mini-macro" }
+serde = "0.7"
```

Comment:
Nope, just for the tests. I can move the tests to serde, as that is where they'd break. That was actually my first intention, then I saw that it compiles in less than 20 seconds on my machine and put it here.